Sources/KeychainSwift.swift

```diff
@@ -95,13 +97,28 @@ open class KeychainSwift {
     let accessible = access?.value ?? KeychainSwiftAccessOptions.defaultOption.value
       
     let prefixedKey = keyWithPrefix(key)
+    var accessWithFlag:Any = accessible as Any
+    
+    var query: [String : Any]
```

Comment:
```suggestion
    let query: [String : Any]
```

Reply:
Query should be `var`, before the function return, query will be updated again.